starter-code/projectMovies/views/celebrities/index.ejs

```diff
@@ -0,0 +1,12 @@
+<h2>CELEBRITIES</h2>
```

Comment:
If you are not using a layout, you need to put all the information (head, body...) in every page.